core/src/main/resources/hudson/PluginWrapper/uninstall.properties

```diff
@@ -1,3 +1,3 @@
-msg=You are about to uninstall the <b>{0}</b> plugin. This will remove the plugin binary from your <code>$JENKINS_HOME</code>, \
-  but it will leave the configuration files of the plugin untouched.
+msg=You are about to uninstall the <b><a href="{1}" target="_blank">{0}</b></a> plugin. This will remove the plugin binary from your <code>$JENKINS_HOME</code>, \
+  but it will leave the configuration files (<code>$JENKINS_HOME/plugins/{2}/</code>) of the plugin untouched.
```

Comment:
This code section points to the locatrion of the unpacked jpi. This doesn't contain any configuration and is removed after uninstallation.